src/main/java/org/folio/services/invoice/InvoicePaymentService.java

```diff
@@ -154,6 +157,6 @@ private boolean isAnyInvoiceLineReleaseEncumbrance(List<InvoiceLine> invoiceLine
   boolean isPaymentStatusUpdateRequired(Map<CompositePoLine, CompositePoLine.PaymentStatus> compositePoLinesWithStatus, CompositePoLine compositePoLine) {
     CompositePoLine.PaymentStatus newPaymentStatus = compositePoLinesWithStatus.get(compositePoLine);
     return (!newPaymentStatus.equals(compositePoLine.getPaymentStatus()) &&
-      !compositePoLine.getPaymentStatus().equals(CompositePoLine.PaymentStatus.ONGOING));
+        !ORDER_LINE_PAYMENT_IGNORED_STATUSES.contains(compositePoLine.getPaymentStatus()));
```

Comment:
formatting: there should be 2 spaces

Reply:
Should it also be 2 spaces for ORDER_LINE_PAYMENT_IGNORED_STATUSES member field?

Reply:
yes for every place, it is our code formatting standard